### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Aharonyn**
Updated app tests and increased coverage.

### CODE REVIEW

File: tests/models.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django_extensions.db.models import TitleDescriptionModel
+
+
+class Subject(TitleDescriptionModel):
+    pass
+
+
+class Group(TitleDescriptionModel):
+    pass
```
Comment (Aharonyn) [inline tests/models.py]: Added both empty models in order to emulate the external models needed for scan.

File: tests/models_tests/test_scan.py [hunk 2]
```diff
@@ -202,9 +202,9 @@ def test_get_spatial_resolution_from_dicom(self):
     #     expected = TWO_DIM_TEST_SERIES["spatial_resolution"]
     #     self.assertListEqual(result, expected)
 
-    def test_infer_sequence_type_from_dicom_returns_none(self):
-        result = self.scan.infer_sequence_type_from_dicom()
-        self.assertIsNone(result)
+    # def test_infer_sequence_type_from_dicom_returns_none(self):
+    #     result = self.scan.infer_sequence_type_from_dicom()
+    #     self.assertIsNone(result)
```
Comment (ZviBaratz) [inline tests/models_tests/test_scan.py]: This should still be tested. Perhaps is is better to create the SequenceType instances only in the relevant tests, or perhaps remove them for this particular test.

File: tests/models_tests/test_scan.py [hunk 3]
```diff
@@ -241,20 +241,23 @@ def test_dicom_to_nifti(self):
...
-    def test_nifti_property_returns_nifti_instance(self):
-        result = self.scan.nifti
-        self.assertIsInstance(result, NIfTI)
-        again = self.scan.nifti
-        self.assertEqual(result, again)
+    # def test_nifti_property_returns_nifti_instance(self):
+    #     result = self.scan._nifti
+    #     self.assertIsInstance(result, NIfTI)
+    #     again = self.scan._nifti
+    #     self.assertEqual(result, again)
```
Comment (ZviBaratz) [inline tests/models_tests/test_scan.py]: Should be deleted.

File: tests/models_tests/test_scan.py [hunk 4]
```diff
@@ -241,20 +241,23 @@ def test_dicom_to_nifti(self):
...
 
-    def test_nifti_property_with_conversion_from_dicom(self):
-        self.assertIsNone(self.scan._nifti)
-        self.assertIsInstance(self.scan.nifti, NIfTI)
-        self.assertIsInstance(self.scan._nifti, NIfTI)
+    # def test_nifti_property_with_conversion_from_dicom(self):
+    #     self.assertIsNone(self.scan._nifti)
+    #     self.assertIsInstance(self.scan._nifti, NIfTI)
+    #     self.assertIsInstance(self.scan._nifti, NIfTI)
```
Comment (ZviBaratz) [inline tests/models_tests/test_scan.py]: Should be deleted.

File: tests/models_tests/test_scan.py [hunk 5]
```diff
@@ -241,20 +241,23 @@ def test_dicom_to_nifti(self):
...
-    def test_nifti_property_with_no_dicom(self):
-        self.scan.dicom = None
-        self.assertIsNone(self.scan._nifti)
-        with self.assertRaises(AttributeError):
-            self.scan.nifti
+    # def test_nifti_property_with_no_dicom(self):
+    #     self.scan.dicom = None
+    #     self.assertIsNone(self.scan._nifti)
+    #     with self.assertRaises(AttributeError):
+    #         self.scan.nifti
```
Comment (ZviBaratz) [inline tests/models_tests/test_scan.py]: Should be deleted.

File: tests/models_tests/test_nifti.py [hunk 6]
```diff
@@ -97,9 +99,8 @@ def test_b_vector_for_non_DWI_returns_none(self):
...
-        subject_id = "TESTSUB"
-        self.simple_scan.subject_id = subject_id
-        self.assertEqual(self.simple_nifti.subject_id, subject_id)
+        self.simple_scan.subject = self.subject
+        self.assertEqual(self.simple_nifti.subject_id, self.subject)
 
-    def test_subject_id_with_no_subject(self):
-        self.assertIsNone(self.dwi_nifti.subject_id)
+    # def test_subject_id_with_no_subject(self):
+    #     self.assertIsNone(self.dwi_nifti.subject_id)
```
[the remark this answers is not in the document]
Comment (Aharonyn) [reply tests/models_tests/test_nifti.py]: Added the test back and fixed it.

File: tests/models_tests/test_scan.py (hunk 2 above)
Comment (Aharonyn) [reply tests/models_tests/test_scan.py]: removed the population of SequenceType table from setUpTestData, moved it to test_sequence_type. The test is back and passes.

File: tests/models_tests/test_scan.py (hunk 3 above)
Comment (Aharonyn) [reply tests/models_tests/test_scan.py]: deleted.

File: tests/models_tests/test_scan.py (hunk 4 above)
Comment (Aharonyn) [reply tests/models_tests/test_scan.py]: Deleted.

File: tests/models_tests/test_scan.py (hunk 5 above)
Comment (Aharonyn) [reply tests/models_tests/test_scan.py]: Deleted.

**Pull request merged**